forum/qa-plugin/ckeditor4/qa-ckeditor4-layer.php

```diff
@@ -69,4 +69,30 @@ function main_part($key, $part)
         }
         qa_html_theme_base::main_part($key, $part);
     }
-}
\ No newline at end of file
+
+    public function head_css()
+    {
+        $this->content['css_src'][] = '/qa-content/css/shCore.css';
+        $this->content['css_src'][] = '/qa-content/css/shThemeDefault.css';
+
+        qa_html_theme_base::head_css();
+    }
+
+    public function head_script()
+    {
+        $scripts = [

```

Comment:
Skoro udało się zgrupować skrypty SyntaxHighlighter'a, to może jest szansa, żeby miejsce ich podpięcia przenieść pod element `.qa-main`, tak jak [tutaj](https://github.com/CodersCommunity/forum.pasja-informatyki.local/pull/208/files#diff-8e56faba9e405a7bed6564bc267a719fR684)? Wtedy strony z tematami (a przynajmniej posty) ładowały by się jeszcze szybciej.

Reply:
Chyba się udało nawet ładnym sposobem, jakbyś mógł zweryfikować czy tak jest w porządku

Reply:
Działa. Można spróbować ustawić skryptom SyntaxHighlightera `defer` i sprawdzić, czy to przyspieszy wczytywanie się strony.